### PR TITLE
GET /me/chats

### PR DESCRIPTION
**Pull request by nkramer**

### CODE REVIEW

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[da666dd](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4235/commits/da666dd135333390a9a722c7b4c9a63cde621c20)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/chat.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/nkramer-me-chats/api-reference/beta/resources/chat.md) | :warning:Warning | | [Details](#user-content-api-reference%2Fbeta%2Fresources%2Fchat.md)
[api-reference/beta/api/chat-list.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/nkramer-me-chats/api-reference/beta/api/chat-list.md) | :white_check_mark:Succeeded | |

<a name="api-reference%2Fbeta%2Fresources%2Fchat.md"></a>
### [api-reference/beta/resources/chat.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/nkramer-me-chats/api-reference/beta/resources/chat.md)
 - **Line 21**: **[Warning]** Invalid file link:(~/beta/api/chat-get.md).
 - **Line 22**: **[Warning]** Invalid file link:(~/beta/api/chat-list-messages.md).
 - **Line 23**: **[Warning]** Invalid file link:(~/beta/api/chat-get-message.md).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C25%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903242357210647-4235%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=IzPVpxNEr8VF5xzuLWByx5GKdyrKWTZtWsuKZlM9Qzk%3D&st=2019-03-24T23%3A55%3A49Z&se=2019-04-25T00%3A00%3A49Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
Docs Build status updates of commit _[faf0fe5](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4235/commits/faf0fe534457223787d0bba3a6e1019ad393ad45)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/chat.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/nkramer-me-chats/api-reference/beta/resources/chat.md) | :warning:Warning | | [Details](#user-content-78e5eaee-719da43c)
[api-reference/beta/api/chat-list.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/nkramer-me-chats/api-reference/beta/api/chat-list.md) | :white_check_mark:Succeeded | |

<a id="78e5eaee-719da43c"></a>
### [api-reference/beta/resources/chat.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/nkramer-me-chats/api-reference/beta/resources/chat.md)
 - **Line 21**: **[Warning]** Invalid file link:(~/beta/api/chat-get.md).
 - **Line 22**: **[Warning]** Invalid file link:(~/beta/api/chat-list-messages.md).
 - **Line 23**: **[Warning]** Invalid file link:(~/beta/api/chat-get-message.md).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C1%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201904011729051723-4235%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=CMlmxjpiAbSgSlL7bfo5uDgATDTITAlgMqR0rTQYb7A%3D&st=2019-04-01T17%3A27%3A50Z&se=2019-05-02T17%3A32%3A50Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: api-reference/beta/api/chat-get-message.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+  "locale": "en-us"
+}
+```
+
+<!-- uuid: 8fcb5dbc-d5aa-4681-8e31-b001d5168d79
+2015-10-25 14:57:30 UTC -->
+<!--
+{
+  "type": "#page.annotation",
+  "description": "Get channel message",
```
Comment (RamjotSingh) [inline api-reference/beta/api/chat-get-message.md]: chat

File: api-reference/beta/api/chat-get-message.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+  "keywords": "",
+  "section": "documentation",
+  "tocPath": "",
+  "suppressions": [
+    "Error: /api-reference/beta/api/channel-get-message.md:\r\n      Exception processing links.\r\n    System.ArgumentException: Link Definition was null. Link text: !INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)\r\n      at ApiDoctor.Validation.DocFile.get_LinkDestinations()\r\n      at ApiDoctor.Validation.DocSet.ValidateLinks(Boolean includeWarnings, String[] relativePathForFiles, IssueLogger issues, Boolean requireFilenameCaseMatch, Boolean printOrphanedFiles)"
```
Comment (RamjotSingh) [inline api-reference/beta/api/chat-get-message.md]: chat

File: api-reference/beta/api/chat-get.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+Do not supply a request body for this method.
+
+## Response
+
+If successful, this method returns a `200 OK` response code and a collection of [chat](../resources/chat.md) objects in the response body.
+
+## Example
+##### Request
+Here is an example of the request.
+<!-- {
```
Comment (RamjotSingh) [inline api-reference/beta/api/chat-get.md]: Fix.

File: api-reference/beta/api/chat-get.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+  "id": "id-value"
+}
+```
+
+<!-- uuid: 8fcb5dbc-d5aa-4681-8e31-b001d5168d79
+2015-10-25 14:57:30 UTC -->
+<!--
+{
+  "type": "#page.annotation",
+  "description": "Get channel message",
```
Comment (RamjotSingh) [inline api-reference/beta/api/chat-get.md]: fix

File: api-reference/beta/api/chat-get.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+  "keywords": "",
+  "section": "documentation",
+  "tocPath": "",
+  "suppressions": [
+    "Error: /api-reference/beta/api/channel-get-message.md:\r\n      Exception processing links.\r\n    System.ArgumentException: Link Definition was null. Link text: !INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)\r\n      at ApiDoctor.Validation.DocFile.get_LinkDestinations()\r\n      at ApiDoctor.Validation.DocSet.ValidateLinks(Boolean includeWarnings, String[] relativePathForFiles, IssueLogger issues, Boolean requireFilenameCaseMatch, Boolean printOrphanedFiles)"
```
Comment (RamjotSingh) [inline api-reference/beta/api/chat-get.md]: chat

Comment (RamjotSingh) [reply api-reference/beta/api/chat-get.md]: Same in other places too.

File: api-reference/beta/resources/chat.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+  "@odata.type": "microsoft.graph.chat"
+}-->
+
+```json
+{
+  "id": "string (identifier)",
+  "topic": "string",
+  "createdDateTime": "dateTimeOffset",
+  "lastUpdatedDateTime": "dateTimeOffset",
+  "chatType": "string"
```
Comment (RamjotSingh) [inline api-reference/beta/resources/chat.md]: Don't think we actually did this property yet. It was deffered to be done when someone else comes along.

File: api-reference/beta/resources/chatmessage.md [hunk 7]
```diff
@@ -20,22 +21,24 @@ Represents an individual chat message within a [channel](channel.md) or chat ent
...
 |from|[identitySet](identityset.md)| Read only. Details of the sender of the message.|
 |etag| string | Version number of the message. |
 |messageType|String|The type of message, current supported values are: message, chatEvent, Typing.|
 |createdDateTime|dateTimeOffset|Read only. Timestamp of when the message was created.|
 |lastModifiedDateTime|dateTimeOffset|Read only. Timestamp of when the message was edited/updated.|
 |deleted|Boolean|Indicates whether a message has been soft deleted.|
 |deletedDateTime|dateTimeOffset|Read only. Timestamp at which the message was deleted, or null if not deleted. |
-|body|[itemBody](itembody.md)|Plaintext/HTML representation of the content of the message. Returns plain text by default, application can choose HTML as part of a query param|
-|summary|string|Summary text of the message that could be used for push notifications and summary views or fall back views|
+|body|[itemBody](itembody.md)|Plaintext/HTML representation of the content of the message. Returns plain text by default, application can choose HTML as part of a query param.|
```
Comment (RamjotSingh) [inline api-reference/beta/resources/chatmessage.md]: contentType property and not query param

File: api-reference/beta/api/chat-get-message.md [hunk 8]
```diff
@@ -0,0 +1,137 @@
+---
+title: "Get chat message"
+description: "Retrieve a single message in a chat."
+author: "nkramer"
+localization_priority: Priority
+ms.prod: "microsoft-teams"
+---
+
+# Get channel message
```
Comment (beauzeaux) [inline api-reference/beta/api/chat-get-message.md]: chat

File: concepts/permissions-reference.md [hunk 9]
```diff
@@ -213,6 +213,22 @@ None.
 
 For more complex scenarios involving multiple permissions, see [Permission scenarios](#permission-scenarios).
 
+## Chats permissions
+
+#### Delegated permissions
+
+|   Permission    |  Display String   |  Description | Admin Consent Required | Microsoft Account supported |
+|:----------------|:------------------|:-------------|:-----------------------|:--------------|
+|_Chat.Read_ |Read your chat messages  |Allows an app to read your 1:1 or group chat messages in Microsoft Teams, on your behalf. |Yes | No |
```
Comment (RamjotSingh) [inline concepts/permissions-reference.md]: Chat.Read does not need Adminconsent.

File: api-reference/beta/api/chat-get.md (hunk 3 above)
Comment (nkramer) [reply api-reference/beta/api/chat-get.md]: done

File: concepts/permissions-reference.md (hunk 9 above)
Comment (nkramer) [reply concepts/permissions-reference.md]: fixed

File: api-reference/beta/resources/chat.md (hunk 6 above)
Comment (nkramer) [reply api-reference/beta/resources/chat.md]: fixed

**Comment by Lauragra** [comment]
@nkramer , are we planning to surface the new APIs in the TOC? I don't see a TOC update with this PR.

**Comment by nkramer** [comment]
TOC added

**Pull request merged**